DesignKit/Source/Colors.swift

```diff
@@ -64,6 +64,12 @@ public protocol Colors {
     /// Background UI color
     var background: ColorType { get }
     
+    /// Global color: The color white.
+    var white: ColorType { get }
+    
+    /// Global color: The EMS brand's purple colour.
+    var ems: ColorType { get }
+    
```

Comment:
I'm semi wondering if these should be grouped in a `theme.globalColors.ems` property or perhaps `theme.colors.global.ems`.